src/analysis/specific_limits.lean

```diff
@@ -845,6 +845,19 @@ begin
   rwa ← le_div_iff (lt_of_le_of_ne (norm_nonneg _) h₁.symm)
 end
 
+/-- A series whose terms are bounded by the terms of a converging geometric series convergences. -/
```

Comment:
```suggestion
/-- A series whose terms are bounded by the terms of a converging geometric series converges. -/
```

Not sure if this is the best; maybe "a subsequence of a converging geometric series also converges"?

Reply:
Also, this kinda uses your lemma from #8022; it's not awful code repetition though